packages/framework/fluid-framework/src/index.ts

```diff
@@ -43,6 +144,30 @@ export * from "@fluidframework/tree";
 
 // ===============================================================
 // Legacy exports
+//
+// To validate that all exports below this point only contribute `@alpha` APIs, and that the above imports only contribute `@public` APIs,
+// comment out these legacy exports, and check that the API report has the same number of `@public` exports and no `@alpha` ones.
+
+export { SharedObject, SharedObjectCore } from "@fluidframework/shared-object-base/internal";
+
+// TODO: these specific exports end up still producing imports, then having those imports reexported.
+export type {
+	EventEmitterEventType,
+	TypedEventEmitter,
+	// It is unclear why ae-forgotten-export requires this since its not referenced.
```

Comment:
TypedEventEmitter references it

Reply:
If you search the API report for it there are no usages so it should not need to be imported. This is entangled with the strangeness that the above two types are not getting inclined in the report like everything else (if that was fixed, then this would be referenced). I'll update the comments to make that more clear.

Reply:
comments have been updated.